benchmarks/benchmarks/stats.py

```diff
@@ -19,4 +18,84 @@ def setup(self, *args):
     def time_anderson_ksamp(self):
         with warnings.catch_warnings():
             warnings.simplefilter('ignore', UserWarning)
-            anderson_ksamp(self.rand)
+            stats.anderson_ksamp(self.rand)
+
+
+class CorrelationFunctions(Benchmark):
+    param_names = ['alternative']
+    params = [
+        ['two-sided', 'less', 'greater']
+    ]
+
+    def setup(self, mode):
+        a = np.random.rand(2,2)*10
+        self.a = a
+
+    def time_fisher_exact(self, alternative):
+        if alternative == 'two-sided':
+            oddsratio, pvalue = stats.fisher_exact(self.a, alternative='two-sided')
+        elif alternative == 'less':
+            oddsratio, pvalue = stats.fisher_exact(self.a, alternative='less')
+        else:
+            oddsratio, pvalue = stats.fisher_exact(self.a, alternative='greater')
+
+
+class InferentialStats(Benchmark):
+    def setup(self):
+        np.random.seed(12345678)
+        self.a = stats.norm.rvs(loc=5, scale=10, size=500)
+        self.b = stats.norm.rvs(loc=8, scale=10, size=20)
+        self.c = stats.norm.rvs(loc=8, scale=20, size=20)
+
+    def time_ttest_ind_same_var(self):
+        # test different sized sample with variances
+        stats.ttest_ind(self.a, self.b)
+        stats.ttest_ind(self.a, self.b, equal_var=False)
+
+    def time_ttest_ind_diff_var(self):
+        # test different sized sample with different variances
+        stats.ttest_ind(self.a, self.c)
+        stats.ttest_ind(self.a, self.c, equal_var=False)
+
+
+class Distribution(Benchmark):
+    param_names = ['distribution', 'properties']    
+    params = [
+        ['cauchy', 'gamma', 'beta'],
+        ['pdf', 'cdf', 'rvs', 'fit']
+    ]
```

Comment:
I quite like how this works. Benchmark output:

               ============== ========= ========= ========== ========
               --                            properties              
               -------------- ---------------------------------------
                distribution     pdf       cdf       rvs       fit   
               ============== ========= ========= ========== ========
                   cauchy      72.14μs   64.87μs   107.18μs   5.88ms 
                   gamma       38.74μs   42.85μs   116.20μs   7.16ms 
                    beta       41.97μs   46.40μs   202.59μs   7.56ms 
               ============== ========= ========= ========== ========